nevergrad/functions/images/core.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from math import sqrt, tan, pi
+import numpy as np
+import nevergrad as ng
+import PIL
```

Comment:
I think you can remove `import PIL` since you import `PIL.Image`

Reply:
Done.